nrt/data/IEEE14/IEEE14_BasicTestCases/IEEE14_LoadVariationRamp/IEEE14.jobs

```diff
@@ -0,0 +1,30 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--
+    Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
+    See AUTHORS.txt
+    All rights reserved.
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, you can obtain one at http://mozilla.org/MPL/2.0/.
+    SPDX-License-Identifier: MPL-2.0
+
+    This file is part of Dynawo, an hybrid C++/Modelica open source time domain
+    simulation tool for power systems.
+--><dyn:jobs xmlns:dyn="http://www.rte-france.com/dynawo">
+  <dyn:job name="IEEE14 - Load Variation">
+    <dyn:solver lib="dynawo_SolverIDA" parFile="solvers.par" parId="2"/>
```

Comment:
do we need to use IDA for this test?

Reply:
There is no difference in the ramp value when using the simplified solver or IDA solver. I just kept the solver used for the IEEE14_LoadVariation test case.